### PR TITLE
Cagrev Update port-forward info under Installing > Verifying

### PR DESCRIPTION
**Pull request by cagrev**

### CODE REVIEW

**Review by dcwebermn — APPROVED** [review]
Approved.  Minor comment - I believe time should be plural in this context - You may have to press enter one more time to get back to the command prompt

File: src/pages/installing/verifying.md [hunk 1]
```diff
@@ -19,21 +19,49 @@ oc get pods --namespace <namespace> | grep controller
...
 
-Check the ACD status by getting the Openshift route and calling the status API.
+Port-forward to the ACD pod retrieved in the previous step. **Note that only an admin can do the port-forward and access the pods through localhost.**
 
 ```
-kubectl get routes --namespace <namespace>
+oc port-forward <pod_name> -n <namespace> 9443:9443 &
+```
+
+The terminal window should return to the command prompt with the port-forward running in the background waiting to accept network traffic on the localhost port 9443. (You may have to press enter one more time to get back to the command prompt.) Note that we used the ampersand (&; a control operator) at the end of the command. The shell assigns a number to the job and displays this job number after a smaller number between brackets. Note the job number returned. Once you are done running the curl command to verify status, you can end the port-forward using that job number.
```
Comment (tpitzen) [inline src/pages/installing/verifying.md:33]: I would say:
"... with the port-forward process running ..."

File: src/pages/installing/verifying.md [hunk 2]
```diff
@@ -19,21 +19,49 @@ oc get pods --namespace <namespace> | grep controller
...
+
+Example output:
+
+```
+$ curl -k 'https://localhost:9443/services/clinical_data_annotator/api/v1/status'
+Handling connection for 9443
+{"version":"2021-06-23T16:41:19Z","upTime":"0d 01:01:19","serviceState":"OK","hostName":"ibm-wh-acd-acd-66bc5786-7rrc7","requestCount":254,"maxMemoryMb":3072,"commitedMemoryMb":3072,"inUseMemoryMb":632,"availableProcessors":16,"concurrentRequests":0,"maxConcurrentRequests":1,"totalRejectedRequests":0,"totalBlockedRequests":0
+```
+
+To end the port-forward job, run the following, where `93053` would be the port-forward job id from the previous example output.
```
Comment (tpitzen) [inline src/pages/installing/verifying.md:57]: You can also mention they can run the "jobs" command to get the job ID.

**Review by tpitzen — APPROVED** [review]

File: src/pages/installing/verifying.md (hunk 1 above)
Comment (cagrev) [reply src/pages/installing/verifying.md:33]: Will update. Thanks.

File: src/pages/installing/verifying.md (hunk 2 above)
Comment (cagrev) [reply src/pages/installing/verifying.md:57]: Will add. Thanks.

**Pull request merged**